### DISCUSSION TITLE
Quarkus native image is unable to pick more than one system env.

### DISCUSSION THREAD

**Discussion by hemjul97** [quarkusio/quarkus#33050] [Community]
I am trying to access system env in application.properties file with ${} but unable to read them. Can anyone help on this? The same is working with non-native.

**Comment by maxandersen** [comment]
can you show what you are doing? what is in the properties file?

**Reply by hemjul97** [reply]
Below is what I have in my config file.

quarkus.application.name=${CONFIG_NAME}
quarkus.spring-cloud-config.enabled=true
# configure the URL where the Config Server listens to HTTP requests - this could have been left out since http://localhost:8888 is the default
quarkus.spring-cloud-config.url=${CONFIG_URI}
quarkus.spring-cloud-config.label=${CONFIG_LABEL}
quarkus.spring-cloud-config.headers.Authorization=Bearer ${CONFIG_ACCESS_TOKEN}
